Fixing deployments test cases

cypo721 opened the pull request:
Fixed both managed and unmanaged deployment test cases for both standalone and domian mode.
Fixed for jbosss7.0.0 DR6 version.

Commit:
Fixed deployments Test Cases for standalone mode

Fixed both managed and unmanaged deployments itest cases for standalone mode

Commit:
Fixed test cases for deployments in domain mode.

Fixed both managed and unmanaged deployments test cases in domain mode.

Commit:
Fixed test cases for domian mode for DR6.

Fixed both managed and unmanaged test  cases for domain mode of jboss 7.0.0 DR6.

Commit:
Fixed test cases for standalone mode of jboss7.0.0 DR6 version.

Fixed both managed and unmanaged test cases for standalone mode of jbos7.0.0 DR6version.

mkrajcov commented on src/main/java/org/jboss/hal/testsuite/page/config/ConfigurationPage.java:
```diff
@@ -40,11 +40,19 @@ public void remove() {
         }
     }
 
+    public void unassign() {
```
Please move this to DeploymentPage as it is specific for deployments only

mkrajcov commented on src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DomainManagedDeploymentsTestCase.java:
```diff
@@ -57,7 +62,8 @@ public static void cleanUp() {
     @Before
     public void before() {
-        browser.navigate().refresh();
+        Graphene.goTo(HomePage.class);
```
You can use Console.withBrowser(browser).refreshAndNavigate method to minimize redundancy in &#64; Before

Commit:
Refactoring of code in deployments test cases.

Removed unneeded try-catch blocks.
Replaced unassign method.
Updated before method in test cases.

zebrik commented:
Can you please remove unused imports? 
I see some in
org.jboss.hal.testsuite.page.runtime.DomainDeploymentPage
org.jboss.hal.testsuite.page.runtime.DeploymentPage
as well as in all testcases in org.jboss.hal.testsuite.test.runtime.deployments package

Commit:
Deleted unused imports and lines.

Deleted unused imports from deployments test cases package and DomainDeploymentPage and DeploymentPage.
